### PR TITLE
fix(menu): force tall bottom menus to render inside the viewport

### PR DESCRIPTION
**Pull request by Kiarokh**
fix: https://github.com/Lundalogik/crm-feature/issues/1225

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/menu/menu.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@ $menu-item-background-color: #ebebeb;
 
 .menu__trigger {
     border-color: transparent;
-    border-width: pxToRem(1);
+    border-width: 1px;
```
Comment (adrianschmidt) [inline src/components/menu/menu.scss:15]: This should be a separate commit from the other change.

Comment (adrianschmidt) [reply src/components/menu/menu.scss:15]: Ping me if you need me to split it up for you 🙂

Comment (Kiarokh) [reply src/components/menu/menu.scss:15]: tnx. I fixed and pushed @adrianschmidt
